crates/cli/src/lib.rs

```diff
@@ -69,24 +69,53 @@ impl Args {
             Command::Start { config: config_path } => {
                 let config = CommitBoostConfig::from_file(&config_path);
 
+                // Initialize Docker client
+                let docker = bollard::Docker::connect_with_local_defaults().expect("Failed to connect to Docker");
```

Comment:
Here, we currently expect that CB and the docker engine run on the same machine. 

This is, however, the way we connect to the docker daemon is modifiable if more complex needs arise: [ref](https://github.com/fussybeaver/bollard/tree/master?tab=readme-ov-file#connecting-with-the-docker-daemon).